actionpack/test/dispatch/session/test_session_test.rb

```diff
@@ -62,4 +62,9 @@ def test_fetch_returns_block_value
     session = ActionController::TestSession.new(one: "1")
     assert_equal(2, session.fetch("2") { |key| key.to_i })
   end
+
+  def test_session_id
+    session = ActionController::TestSession.new
+    assert_instance_of String, session.id.public_id
```

Comment:
Should we test `session["session_id"]`?

Reply:
I added test for `session["session_id"]`